### PR TITLE
refactor(sql translator): update query transformation method & complete filter step

### PR DESCRIPTION
**Pull request by raphaelvignes**
- Refactor a bit the data structure forwarded between steps (an `SQLQuery` object) Laputa will need to be updated accordingly
- Replicate all conditions possible from `pipeline_executor`

### CODE REVIEW

**Comment by raphaelvignes** [comment]
[(5) [Snowflake VQB Translator] - As An App Maker I can filter using snowflake vqb](https://trello.com/c/H7tzJFOw/1088-5-snowflake-vqb-translator-as-an-app-maker-i-can-filter-using-snowflake-vqb)

**Comment by raphaelvignes** [comment]
[(5) [Snowflake VQB Translator] - As An App Maker I can filter using snowflake vqb [4]](https://trello.com/c/H7tzJFOw/1088-5-snowflake-vqb-translator-as-an-app-maker-i-can-filter-using-snowflake-vqb-4)

**Review by f-bb-toucantoco — APPROVED** [review]
A première vue je n'ai rien à dire
Je ne maitrise pas encore beaucoup le composant, la review de Fred / Luc / David me permettra de voir si je me suis planté

File: server/weaverbird/backends/sql_translator/sql_pipeline_translator.py [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
 
 def translate_pipeline(
-    pipeline_to_translate: Pipeline, sql_table_retriever: SQLTableRetriever
+    pipeline_to_translate: Pipeline,
+    sql_query_retriever: SQLTableRetriever,  # TODO : rename it to sql_table_retriever when ready
```
Comment (fspot) [inline server/weaverbird/backends/sql_translator/sql_pipeline_translator.py:12]: Du coup il faudrait peut-être également revert le nom de la classe ?

**Review by fspot — APPROVED** [review]
lgtm :ok_hand:

File: server/weaverbird/backends/sql_translator/sql_pipeline_translator.py (hunk 1 above)
Comment (raphaelvignes) [reply server/weaverbird/backends/sql_translator/sql_pipeline_translator.py:12]: Oki je fais ça en attendant de changer l'input :+1:

**Pull request merged**